core/src/main/java/com/amplifyframework/core/DeveloperMenuActivity.java

```diff
@@ -74,4 +122,30 @@ private void changeVisibility() {
             devMenuLayout.setVisibility(View.VISIBLE);
         }
     }
+
+    /**
+     * Hides the main developer menu view, displays the given View, and
+     * changes the title text on the action bar.
+     * @param newView View to display.
+     * @param titleTextId ID of the string resource for the action bar title text.
+     */
+    private void leaveMainView(View newView, int titleTextId) {
```

Comment:
It's a subtle point here, but the `Activity` ought to be about visual display. So I would phrase this as "display" and "hide" the main view, instead of the navigation-sounding (control) names you're using.